docs/pages/blog/mui-x-v5.md

```diff
@@ -0,0 +1,191 @@
+
+In previous versions most of the various `DataGrid` and `DataGridPro` components had a CSS specificity of 2, meaning that style overrides and customizations were harder, requiring you to look at the DOM tree in order to pick the correct selector.
+With MUI X v5 we have reduced the CSS specificity of most of the internal `DataGrid` and `DataGridPro` components to 1.
+This will enable developers to more easily change the look and feel of the grid's components.
+
+**Before**
+
+```jsx
+const useStyles = makeStyles(() => ({
+  root: {
+    '& .MuiDataGrid-toolbarContainer': {
+      padding: 50,
+    },
+  },
+}));
+
+const MyCustomToolbar = () => {
+  return <GridToolbarContainer>My custom toolbar</GridToolbarContainer>;
+};
+
+export default function App() {
+  const classes = useStyles();
+
+  return (
+    <div style={{ height: 400, width: '100%' }} className={classes.root}>
+      <DataGrid
+        components={{
+          Toolbar: MyCustomToolbar,
+        }}
+      />
+    </div>
+  );
+}
+```
+
+**After**
+
+```jsx
+const useStyles = makeStyles(() => ({
+  root: {
+    padding: 50,
+  },
+}));
+
+const MyCustomToolbar = () => {
+  const classes = useStyles();
+
+  return (
+    <GridToolbarContainer className={classes.root}>
+      My custom toolbar
+    </GridToolbarContainer>
+  );
+};
+
+export default function App() {
+  return (
+    <div style={{ height: 400, width: '100%' }}>
+      <DataGrid
+        components={{
+          Toolbar: MyCustomToolbar,
+        }}
+      />
+    </div>
+  );
+}
+```
+
+#### Limitations
+
+Although this was a clear improvement, we still had to keep a style specificity of 2 for some parts of the `DataGrid` and `DataGridPro`, specifically the `GridColumnHeaderItem`, `GridRow` and `GridCell` along with all of the components that are nested in them.
+This is due to performance implications related to how Emotion injects styles into the page,
+and was necessary to keep the performance of our virtualization engine at its optimal.
+
+## v4 migration
+
+We strongly recommend you migrate MUI X to v5. In MUI X v5 we have not only added additional features but also made significant internal improvements and performance optimizations that won't be included in v4. All new `DataGrid` and `DataGridPro` features will be only available in MUI X v5.
+You can check [this page](https://mui.com/components/data-grid/migration-v4/) to learn more about migrating to v5.
+
+## What's next?
+
+More exciting things are coming! We have big plans for the rest of this quarter in terms of features we expect to release in both the `DataGrid` and `DataGridPro` components. Features such as **tree data**, **column pinning**, and **variable row height** are part of our roadmap.
+
+### Public roadmap
+
+You can view our public roadmap on GitHub to learn about what features we're working on, what stage they're at, and when we expect to bring them to you:
+
+- [MUI X](https://github.com/mui-org/material-ui-x/projects/1)
```

Comment:
A header and a ul > li feel alone here. If we don't plan to add more content, we can remove them

Reply:
I think that we could share our longer-term plan in this section, we talk 2021 only. We could share what we have in mind before v6. I think that a bias on the why can help. It's then easier for developers to emphasize with us, and so we don't have to provide too many specific details (that we might not do, changing our mind)

Reply:
Noted - I reworked the sentence a but to remove the `ul > li`.
Regarding expanding the section with our long-term goal - I would love to do that but I don't think we have discussed the long-term goal with the team so maybe it's better to first define it between ourselves.